### ISSUE TITLE
Inner struct causing an incorrect ending brace error

### ISSUE THREAD

**Issue by krzys-h** [colobot/colobot-lint#12]
Latest @tomaszkax86's code (from gldevice33.h):
```C++
struct UniformLocations
{
    // Uniforms
    //! Projection matrix
    GLint projectionMatrix = -1;
    //! View matrix
    GLint viewMatrix = -1;
    //! Model matrix
    GLint modelMatrix = -1;
    //! Shadow matrix
    GLint shadowMatrix = -1;
    //! Normal matrix
    GLint normalMatrix = -1;

    //! Primary texture sampler
    GLint primaryTexture = -1;
    //! Secondary texture sampler
    GLint secondaryTexture = -1;
    //! Shadow texture sampler
    GLint shadowTexture = -1;

    //! true enables texture
    GLint textureEnabled[3] = {};

    // Alpha test parameters
    //! true enables alpha test
    GLint alphaTestEnabled = -1;
    //! Alpha test reference value
    GLint alphaReference = -1;

    //! true enables fog
    GLint fogEnabled = -1;
    //! Fog range
    GLint fogRange = -1;
    //! Fog color
    GLint fogColor = -1;

    //! Shadow color
    GLint shadowColor = -1;

    //! true enables lighting
    GLint lightingEnabled = -1;
    //! Ambient color
    GLint ambientColor = -1;
    //! Diffuse color
    GLint diffuseColor = -1;
    //! Specular color
    GLint specularColor = -1;

    struct LightLocations
    {
        //! true enables light
        GLint enabled = -1;
        //! Light type
        GLint type = -1;
        //! Position or direction vector
        GLint position = -1;
        //! Ambient color
        GLint ambient = -1;
        //! Diffuse color
        GLint diffuse = -1;
        //! Specular color
        GLint specular = -1;
        //! Attenuation
        GLint attenuation = -1;
    } lights[8];
};
```

colobot-lint throws an ```"Body of declaration or statement ends in a style that is not allowed"``` error at both beginning and end of LightLocations struct declaration. To my understanding, this code is not violating code style in any way.

**Comment by piotrdz** [comment]
I'll look into that but I think the problem is connected with how Clang handles such declarations.
Basically this is a declaration and a definition in the same line and this is what I specifically look for in `BlockPlacementRule`.
Also, you'll find another rule that marks anonymous structs and unions as issues in `ClassNamingRule`.

**Comment by piotrdz** [comment]
Fixed in b4b70e8.
